### DISCUSSION TITLE
Governor: Increase Daily Notional Limits on Ethereum and Solana

### DISCUSSION THREAD

**Discussion by mizdler** [wormhole-foundation/wormhole#4163] [Governance] [answered]
With demand for bridging at an all-time high, Token Bridge is consistently reaching the daily notional limits on Ethereum and Solana. This is evident from recent activity data, where users are encountering restrictions due to these caps:

![image](https://github.com/user-attachments/assets/eab9b6f3-28d9-495e-9d1b-c8ad0884966b)

To accommodate growing user demand and ensure a seamless bridging experience, I propose increasing the daily notional limits as follows:

- Ethereum: Increase from 50M to 100M
- Solana: Increase from 25M limit to 50M

**Comment by johnsaigle** [comment, marked as answer]
These changes have been applied.
